### PR TITLE
Register the commands as services.

### PR DESCRIPTION
**Pull request by mvrhov**
This is needed since sf4

### CODE REVIEW

**Comment by Nyholm** [comment]
Awesome.

File: Resources/config/console.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+services:
+    php_translator.console.delete_obsolete:
+        class: Translation\Bundle\Command\DeleteObsoleteCommand
+        public: false
```
Comment (Nyholm) [inline Resources/config/console.yml]: Commands must be public

Comment (mvrhov) [reply Resources/config/console.yml]: Not since >3.4. Everything below automatically registers commands

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**